modules/auxiliary/scanner/http/cisco_ironport_enum.rb

```diff
@@ -61,6 +61,7 @@ def check_conn?
         'method'    => 'GET'
       })
       print_good("#{rhost}:#{rport} - Server is responsive...")
+      return 1
     rescue ::Rex::ConnectionRefused, ::Rex::HostUnreachable, ::Rex::ConnectionTimeout, ::Rex::ConnectionError, ::Errno::EPIPE
       return
```

Comment:
Good point @wvu-r7. I just took the fact that it prints "Server is responsive..." as a sign that everything is fine and return 1.  Returning res may be a better choice.  Thanks.